staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -720,7 +726,7 @@ func (o *GetOptions) watch(f cmdutil.Factory, cmd *cobra.Command, args []string)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	intr := interrupt.New(nil, cancel)
-	intr.Run(func() error {
+	return intr.Run(func() error {
```

Comment:
This too was returning an error until I added `err == watchtools.ErrWatchClosed` below. So many ignored errors (I haven't fixed them all), I wonder why linters don't pick them up.